### DISCUSSION TITLE
plugins  vi-xournalpp

### DISCUSSION THREAD

**Discussion by vladQll** [xournalpp/xournalpp#4628] [Q&A]
hello, what's up @raw-bacon I wanted to know if your plugin can also be used in windows, and if it is necessary to install vim, I tried to download it as zip but windows blocks it because it recognizes it as a virus.
![image](https://user-images.githubusercontent.com/28522011/215346781-2ac1cfbb-296a-4043-bddf-528755fac7ba.png)

**Comment by lryffel** [comment]
Hi,

I have no way to check, but I expect the plugin to work on Windows as well. It's probably easiest if you install it by cloning the repo into `C:\Users\<user>\AppData\Local\xournalpp\plugins` (assuming you're running a current version, the plugin will only work on 1.1 and above anyway), then enable it through the plugins GUI within Xournal++. The plugin does not depend on vim.

As for why Windows thinks the plugin is a trojan I have no idea, the whole repo contains no executables. Are you sure you downloaded from the [correct repo](https://github.com/raw-bacon/vi-xournalpp)?
